content-repo/extra-docs/articles/tim-edl.md

```diff
@@ -1,164 +1,190 @@
 ---
 id: tim-edl
 title: Add Indicators to SIEM
-description: This article walks you through setting up a playbook to take indicators from a threat intel feed, enrich the indicators, and push them to your SIEM.
+description: This article takes you through setting up a playbook, which fetches indicators from a threat intel feed, enriches the indicators, and then pushes them to your SIEM.
 ---
-This article walks you through setting up a playbook to take indicators from a threat intel feed, enrich the indicators, and push them to your SIEM.
+In Cortex XSOAR, you can set up a playbook to fetch indicators from a threat intel feed, enrich the indicators, and push them to your SIEM. 
 
-1. Define a threat intel feed to ingest indicators to your system.
+Although this article takes you through this process, it is not intended to cover every possible scenario. It provides a practical scenario that touches on important points from which you can use in other scenarios. You should  follow these steps.
 
-2. Customize a playbook that is triggered by the feed to process the indicators and determine which are legitimate.
+1. [Configure the threat intel feed](#configure-the-threat-intel-feed): Enables you to ingest indicators into Cortex XSOAR.
+2. [Create a list of indicators not to process](#create-a-list-of-indicators-not-to-process): Create a list for indicators to exclude, which will be added to the playbook. 
 
-3. Define a feed-triggered job that runs every time indicators are ingested by the feed.
+3. [Customize the playbooks](#customize-your-playbooks): These playbooks process the indicators and determine which are legitimate.
 
-4. Review the indicators and determine with which tags each indicator should be tagged.
+4. [Define a feed-triggered job](#define-a-feed-triggered-job): Run a job every time indicators are ingested by the feed. 
```

Comment:
Don't users need to add a job so that the playbook is triggered? This is what we say here: https://xsoar.pan.dev/docs/reference/playbooks/tim---process-indicators---manual-review